src/openforms/appointments/admin.py

```diff
@@ -73,17 +74,22 @@ def get_list_display(self, request):
             list_display.remove("get_object_actions")
         return list_display
 
-    def get_object_actions(self, obj) -> str:
+    def get_object_actions(self, obj: AppointmentInfo) -> str:
         if not obj.status == AppointmentDetailsStatus.success:
             return "-"
 
         if obj.start_time and obj.start_time <= timezone.now():
             return "-"
 
-        actions = [
-            ("change", _("Change")),
-            ("cancel", _("Cancel")),
+        actions: list[tuple[Literal["cancel", "change"], str]] = [
+            ("cancel", gettext("Cancel")),
         ]
```

Comment:
type checker was complaining as it infers the `str` of `tuple[str, ...]` to be a `Literal` :(